.github/workflows/build.yml

```diff
@@ -91,63 +92,107 @@ jobs:
         if: github.event_name == 'release'
         run: poetry publish -u __token__ -p ${{ secrets.PYPI_TOKEN }}
 
-  windows:
+  gvsbuild:
```

Comment:
I'd rather have the build step be named windows. This makes more sense when looking at the build results.

Reply:
Hi @amolenaar, thanks for the feedback :+1:.

I split the windows build in to two steps:

1. Build GTK for Windows, or grab it from cache if it is already built
2. Everything else (install dependencies, test, package)

This was the first step that was renamed to gvsbuild, but I just renamed it to `windows-gtk-build`, hopefully that is more clear.

I am also working on packaging GTK for Windows and putting it on [winget](https://docs.microsoft.com/en-us/windows/package-manager/winget/) (Microsoft's new Chocolately packaging system that will be built in to Windows). So hopefully we can get rid of step 1 in the near future completely. I created a repo to work on that here: https://github.com/gaphor/gtk-win-installers